livingatlas/pipelines/src/test/java/au/org/ala/pipelines/interpreters/AlaLocationInterpreterTest.java

```diff
@@ -364,6 +365,24 @@ public void assertCountryCentre() {
     assertEquals(Country.AUSTRALIA.getTitle(), lr.getCountry());
   }
 
+  /**
+   * List countries not in coutrycentrefile
+   *
+   * @throws IOException
+   */
+  @Test
+  public void countryNameMatch() throws IOException {
+    Set<String> countryInFile = countryCentrePoints.keys();
+
+    for (Country country : Country.values()) {
+      String name = country.getTitle().toUpperCase();
+      String countryCode = country.getIso2LetterCode().toUpperCase();
+      if (!countryInFile.contains(name)) {
+        System.out.println(countryCode + " : " + country.getTitle());
```

Comment:
can we replace with log.info or log.debug ?

Reply:
Done